src/pqueue.js

```diff
@@ -0,0 +1,102 @@
+// 'use strict'
+//
+// class Node {
+//   constructor ( value, priority ){
+//     this.arr = []
+//     this.value = value
+//     this.priority = priority
+//   }
+// }
+
+export default class PriorityQueue {
+  constructor(value) {
+    this.arr = value || []
+    this.index = 0
+  }
+
+  enqueue(data, priority) {
+    return this.arr[this.index++] = {data, priority}
+  }
+
+  front() {
+    let current,
```

Comment:
It is considered best practice to have consistently indented code at 2 spaces.  Aligning indented variables in this way requires all future developers to space space space space space space space space space space space space space space space space in order to maintain consistency in the code.  If that was annoying to read, think of how annoying it would be to type.  😄